app/models/micropost.rb

```diff
@@ -1,2 +1,10 @@
 class Micropost < ApplicationRecord
+    belongs_to :user
+
+    validates :content, length: {
+        minimum: 3,
+        maximum: 300
```

Comment:
If I'm not mistaken, `maximum` was `140`.

Reply:
> Please, resolve the problem about committing `.idea` files.

I don't have .idea files in my project, maybe you're mistaken. I use vscode

Reply:
> If I'm not mistaken, `maximum` was `140`.

Ok, I'll change it